docs/specs/README.md

```diff
@@ -0,0 +1,64 @@
+# Guidelines for Feature Request(s)
+
+This document covers, how to contribute new feature(s) or significant design changes to BigchainDB. Since, contribution
+of such functionality involves design discussion and might take multiple releases to complete,
+we expect the contributors to follow certain guidelines before proceeding with the implementation.
+
+
+## Classification
+How to classify your contribution as a feature and determine if it needs design discussion. 
+
+Here are some baseline criterion that can help you classify your idea, as a feature:
+
+- Requires significant effort/changes to the current BigchainDB implementation and it is not self
+ contained (e.g. API changes,  introducing a new component, takes more than 20 Wrigleys
+ (Wrigley is a `man day` @ BigchainDB))
```

Comment:
is a 'man day' a work day? if so it's better to call it work day.

Reply:
`man day` is a more common term.

```
man-day - noun

a day regarded in terms of the amount of work that can be done by one person within this period.
```

Reply:
Also think "work day" is more clear, or why not "dev day"? Rarely a need attaching a gender to a time unit, more suggestions here: [What is a gender-neutral alternative to the expression “man-days”?](https://english.stackexchange.com/questions/109761/what-is-a-gender-neutral-alternative-to-the-expression-man-days)

Also, because it's marked up as code I was totally reading it as opening the `man` page of the `day` executable. Using any other suggestion would also solve that :-)

Reply:
I guess `dev days` would be a better alternative, in that case. `work day` is a [company](https://www.workday.com/).